src/display.cpp

```diff
@@ -375,7 +375,7 @@ void dp_refresh(bool nextPage) {
     dp_clear();
     break;
 #else // skip this page
-    DisplayPage++;
+    DisplayPage = 0;
```

Comment:
This seems like a hack aorund a bug to me.  I think the bug is in _display.h_:
`#define DISPLAY_PAGES (7) // number of paxcounter display pages
`Please try with (6). That should work.